### PR TITLE
[ADD] purchase_requisition: Migration scripts

### PR DESCRIPTION
**Pull request by pedrobaeza**

### CODE REVIEW

File: addons/purchase_requisition/migrations/8.0.0.1/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                insert into wkf_workitem
+                (id, act_id, inst_id, state)
+                VALUES (%s, %s, %s, 'active')
+                """, (id_new, activities[requisition.state], instance_id))
+
+
+@openupgrade.migrate()
+def migrate(cr, version):
+    pool = pooler.get_pool(cr.dbname)
+    openupgrade_80.set_message_last_post(cr, SUPERUSER_ID, pool, ['crm.claim'])
```
Comment (JordiBForgeFlow) [inline addons/purchase_requisition/migrations/8.0.0.1/post-migration.py]: change from 'crm.claim' to 'purchase.requisition'.

**Comment by pedrobaeza** [comment]
@jbeficent, changed!

File: addons/purchase_requisition/migrations/8.0.0.1/openupgrade_analysis_work.txt [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+purchase_requisition / product.template         / purchase_requisition (boolean): NEW
+# Field moved
+
+purchase_requisition / purchase.order.line      / quantity_bid (float)          : NEW
+# Duplicated value from current purchase order line quantity
+
+purchase_requisition / purchase.requisition     / account_analytic_id (many2one): NEW relation: account.analytic.account
+purchase_requisition / purchase.requisition     / multiple_rfq_per_supplier (boolean): NEW
+purchase_requisition / purchase.requisition     / ordering_date (date)          : NEW
+purchase_requisition / purchase.requisition     / procurement_id (many2one)     : NEW relation: procurement.order
```
Comment (StefanRijnhart) [inline addons/purchase_requisition/migrations/8.0.0.1/openupgrade_analysis_work.txt]: This is the counterpart many2one of the existing many2one 'requisition_id' from the procurement to the requisition table. So you can fill this new 'procurement_id' field from here.

**Comment by StefanRijnhart** [comment]
Can you forceload this changed noupdate data?

```
    <record id="group_purchase_requisition_user" model="res.groups">
        <field name="implied_ids" eval="[(3, ref('purchase.group_purchase_user'))]"/>
    </record>
```

**Comment by StefanRijnhart** [comment]
Thanks, I made some suggestions.

**Comment by pedrobaeza** [comment]
@StefanRijnhart, changes done.

**Comment by StefanRijnhart** [comment]
Thanks! :+1:

**Comment by StefanRijnhart** [comment]
@jbeficent did you test these scripts by any chance? Looking for another thumbs-up

**Comment by JordiBForgeFlow** [comment]
I checked on a migrated db and looks good. :+1:

**Comment by StefanRijnhart** [comment]
Thanks for testing!

**Pull request merged**